### DISCUSSION TITLE
Unable to connect to mobile Coinbase Wallet

### DISCUSSION THREAD

**Discussion by HowdyChad** [WalletConnect/walletconnect-monorepo#2271] [Developer Support]
When trying to scan from https://web3modal.com/ I get an error in the mobile Coinbase wallet. Lot of sites use this wallet connect, but the coinbase self custody mobile wallet doesn't recognize the QR code. It prints an error, but the most important part is cutoff. Is there a way get coinbase wallet to work with walletconnect? I don't want to install the coinbase web extension.

**Comment by xzilja** [comment]
Coinbase doesn't use WalletConnect, so to enable it you'll need to add their wagmi connector which displays their proprietary qrcode

**Reply by crypblizz8** [reply]
Thanks @0xAsimetriq, @kewlniss here is [extra documentation](https://wagmi.sh/examples/connect-wallet) from wagmi that should help you out regarding the Coinbase wallet integration.
